contrib/utils/libmesh-dbg.pc.in

```diff
@@ -8,9 +8,8 @@ Description: The libMesh Finite Element Library
 Requires: @libmesh_pkgconfig_requires@
 Version: @VERSION@
 Libs: -Wl,-rpath,${libdir} -L${libdir} -lmesh_dbg -ltimpi_dbg \
-      @libmesh_installed_LIBS@ @libmesh_optional_LIBS@      
-Libs.private: 
+      @libmesh_installed_LIBS@ @libmesh_optional_LIBS@
+Libs.private:
 Cflags: @CPPFLAGS_DBG@ @CXXFLAGS_DBG@ \
-        -I${includedir} -I${includedir}/contrib \
```

Comment:
Huh.  When did we ever install header files to ${includedir}/contrib?  Thanks for catching this.

Reply:
I have no idea - I might be the only person who has tried to interface with libMesh via pkgconfig in awhile :)